DESCRIPTION

```diff
@@ -33,7 +34,8 @@ Suggests:
     rmarkdown,
     usethis,
     vctrs (>= 0.1.0),
-    xml2
+    xml2,
+    testthat (>= 2.1.0)
```

Comment:
This may not hurt anything, but it is kind of weird for testthat to Suggest itself.

Reply:
Ooops, I guess we need to special case `use_test()`